### PR TITLE
etcdserver: rename defaultCompactionSleepInterval var

### PR DESCRIPTION
**Pull request by JalinWang**
fix comment in PR https://github.com/etcd-io/etcd/pull/13018#discussion_r1730275709

### CODE REVIEW

**Comment by ivanvc** [comment]
Thanks for the pull request, @JalinWang. It looks like you missed updating the value in `server/storage/mvcc/kvstore_test.go`. Can you please update it, too?

@ahrtr, you suggested the variable be named `defaultBatchInterval`. Are you okay with `defaultCompactionSleepInterval`?

**Comment by JalinWang** [comment]
> Thanks for the pull request, @JalinWang. It looks like you missed updating the value in `server/storage/mvcc/kvstore_test.go`. Can you please update it, too?

Apologies :( I overlooked its usage in later commits. 

> @ahrtr, you suggested the variable be named `defaultBatchInterval`. Are you okay with `defaultCompactionSleepInterval`?

Following `cfg.CompactionBatchLimit = defaultCompactBatchLimit`, I think it's better to use ` defaultCompactionSleepInterval` instead of the suggested `defaultBatchInterval` for `cfg.CompactionSleepInterval`.

**Comment by ahrtr** [comment]
> update: emmm `cfg.Compact**ion**BatchLimit -> defaultCompactBatchLimit`... Should I make them the same? 🫤

OK to me.

**Review by serathius — APPROVED** [review]

File: server/storage/mvcc/kvstore.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ var (
 
 var restoreChunkKeys = 10000 // non-const for testing
 var defaultCompactBatchLimit = 1000
```
Comment (ahrtr) [inline server/storage/mvcc/kvstore.go:42]: nit as you suggested
```suggestion
var defaultCompactionBatchLimit = 1000
```

Comment (JalinWang) [reply server/storage/mvcc/kvstore.go]: 👌  done

**Comment by ahrtr** [comment]
> Please squash the commits.

Let me do it this time.

**Review by ahrtr — APPROVED** [review]

**Pull request merged**